### ISSUE TITLE
爱可可微博热门分享(3.22)

### ISSUE THREAD

**Issue by fly51fly** [fly51fly/aicoco#107]
No 1. <a href="https://weibo.com/1402400261/O68BEFK2R">#免费##抽奖# 携手@图灵文化 @图灵新知 送出3本《hello 算法》</a>
No 2. <a href="https://weibo.com/1402400261/O68z3vsYr">#免费##抽奖# 携手@博文视点Broadview 送出3本《大语言模型：原理与工程实践(全彩)》</a>
No 3. <a href="https://weibo.com/1402400261/O68Jg0DkB">SceneScript：语言模型赋能3D场景理解</a>
No 4. <a href="https://weibo.com/1402400261/O6aMsh6ew">TorchTune：易于使用的 PyTorch 库，可轻松编写、微调和实验LLM模型。该库提供了多种功能，包括使用 native-PyTorch 实现的流行语言模型，支持各种格式的复原，以及提供训练和评估工具，例如 HF 格式的检查点支持</a>
No 5. <a href="https://weibo.com/1402400261/O6aAR2rFQ">qlora-pipe：用于训练大语言模型的开源脚本，可以在四块4090 GPU上对LLM进行定制训练</a>
No 6. <a href="https://weibo.com/1402400261/O6aOecw1v">Devika：基于开源的 AI 软件工程师，可以理解人类的高级指令，并根据这些指令，分解成步骤，进行相关信息的研究，并编写代码实现目标</a>
No 7. <a href="https://weibo.com/1402400261/O6b49eEqP">SpatialData: 用于处理多模态空间 omics数据的开源框架</a>
No 8. <a href="https://weibo.com/1402400261/O67EX9Ewx">[CL]《Encode Once and Decode in Parallel: Efficient Transformer Decoding》</a>
No 9. <a href="https://weibo.com/1402400261/O6aPq4XB3">Ludic：轻量Python框架，类似于 React，使用组件的方式构建 HTML 页面。它与 htmx.org框架集成在一起，无需编写大量 JavaScript，可以与 Starlette框架一起使用</a>
No 10. <a href="https://weibo.com/1402400261/O6b5z0qtw">几篇论文实现代码</a>
No 11. <a href="https://weibo.com/1402400261/O6b2509HT">RWKV_Pytorch：用纯Pytorch原生实现的RWKV大语言模型的推理框架</a>
No 12. <a href="https://weibo.com/1402400261/O61z7renB">GAMA-Bench：多Agent环境下LLM博弈能力的性能测试基准</a>
No 13. <a href="https://weibo.com/1402400261/O69gfhC5j">AGI的定义之争：人工智能的终极目标还有多远？</a>
No 14. <a href="https://weibo.com/1402400261/O658ja6GM">晚安～ [月亮]</a>
No 15. <a href="https://weibo.com/1402400261/O67YsjTA0">[LG] RewardBench: Evaluating Reward Models for Language Modeling ......</a>
No 16. <a href="https://weibo.com/1402400261/O61wG8Lpb">Lightning Thunder：开源 PyTorch 代码编译器，可以使 PyTorch 程序更快，无论是单个加速器还是分布式环境</a>
No 17. <a href="https://weibo.com/1402400261/O5q2nDkz5">ArXiv Paper Reader：旨在简化和流利的arXiv论文阅读，使用 LaTeX 代码转换为 HTML 页面，然后提取文本和公式，将其转换为视频，并创建与 PDF 文档匹配的图，以及文本分段并将其转换为音频</a>
No 18. <a href="https://weibo.com/1402400261/O6aYMeAQs">面向无人驾驶的世界模型相关论文资源列表</a>
No 19. <a href="https://weibo.com/1402400261/O67NpDkRT">[LG] Evaluating Frontier Models for Dangerous Capabilities ......</a>
No 20. <a href="https://weibo.com/1402400261/O6b5ub8q2">ReverserAI：通过使用本地大语言模型 (LLM)，自动推荐函数名称，帮用户进行软件逆向工程</a>
No 21. <a href="https://weibo.com/1402400261/O61EowFIA">'FaceSearchSDK_Android - On Device Android Face 1:N and M:N ......</a>
No 22. <a href="https://weibo.com/1402400261/O680Jvcgj">[CV] RadSplat: Radiance Field-Informed Gaussian Splatting ......</a>
No 23. <a href="https://weibo.com/1402400261/O61xXbnCf">LLaMA Nuts and Bolts：旨在通过代码和文档了解 LLaMA 和其组件运行背后的原理的开源项目</a>
No 24. <a href="https://weibo.com/1402400261/O6aApgRJw">Hugot: 用于 golang 的 huggingface  transformer pipelines，可以轻松地将 Python 中的 huggingface 模型部署到 golang 应用中</a>
No 25. <a href="https://weibo.com/1402400261/O67Et3vgN">[CV]《When Do We Not Need Larger Vision Models?》</a>
No 26. <a href="https://weibo.com/1402400261/O5Ven65Ye">text-splitter：将文本分割成不同的片段，以便在处理更大的文本时更好地处理文本</a>
No 27. <a href="https://weibo.com/1402400261/O5ZdD4U3Z">[good]//这里以通俗易懂的方式来分析Sora的可能做法，包括它的整体结构以及关键组件。我希望即使您不太懂技术，也能大致看明白Sora的可能做法，所以画了几十张图来让看似复杂的机制更好理解，如果您看完对某部分仍不理解，那是我的问题。技术神秘化的去魅：S...</a>
No 28. <a href="https://weibo.com/1402400261/O5V5MDtvv">img2img-turbo：基于SD-Turbo的一步图到图变换</a>
No 29. <a href="https://weibo.com/1402400261/O5Zfw6TaS">Common Corpus 让 AI 更透明</a>
No 30. <a href="https://weibo.com/1402400261/O67Cr6h5t">该论文通过逆向训练语言模型并保留实体词序不变，大幅减轻了大型语言模型中的“颠倒诅咒”，是解决该问题的有效方法之一。//[CL]《Reverse Training to Nurse the Reversal Curse》</a>
No 31. <a href="https://weibo.com/1402400261/O61ud4TmF">大语言模型与工具使用相关论文资源列表</a>
No 32. <a href="https://weibo.com/1402400261/O5Yp69O30">[LG]《Non-negative Contrastive Learning》</a>
No 33. <a href="https://weibo.com/1402400261/O67Rq1hAm">[LG] Evolutionary Optimization of Model Merging Recipes 网页链接 ......</a>
No 34. <a href="https://weibo.com/1402400261/O5Zrf9g14">李世乭“人机大战”8年后的反思和感悟</a>
No 35. <a href="https://weibo.com/1402400261/O67Ey0uIn">通过在多个图像尺度上提取特征，不扩大模型规模也可以获得强大的视觉表示，在各类下游任务上体现出优越性。//[CV]《When Do We Not Need Larger Vision Models?》</a>
No 36. <a href="https://weibo.com/1402400261/O67BqCspO">早！[太阳] ​</a>
No 37. <a href="https://weibo.com/1402400261/O5ZlXunOm">AnimateDiff-Lightning：超快的文生视频实现</a>
No 38. <a href="https://weibo.com/1402400261/O5FCWiOLD">[LG] Understanding the Double Descent Phenomenon in Deep Learning ......</a>
No 39. <a href="https://weibo.com/1402400261/O61rC2jl3">GritQL 是一个描述式查询语言，可以用于搜索和修改代码，支持多种编程语言</a>
No 40. <a href="https://weibo.com/1402400261/O61DBbZDs">OpenDMD：基于分布匹配蒸馏的一步扩散的开源实现和模型</a>
No 41. <a href="https://weibo.com/1402400261/O6aBRuVBB">Pico MLX Server：Apple 的 MLX AI框架的轻松入门方式，提供了一个用于 MLX Server的GUI</a>
No 42. <a href="https://weibo.com/1402400261/O5FFDAV2O">[CV] VideoAgent: Long-form Video Understanding with Large ......</a>
No 43. <a href="https://weibo.com/1402400261/O5ZkH1crh">用GaLore在消费级硬件上训练大模型</a>
No 44. <a href="https://weibo.com/1402400261/O5LCjCzTu">大模型安全相关阅读列表</a>
No 45. <a href="https://weibo.com/1402400261/O61CxpYtG">用于用户建模的大型语言模型(LLM-UM)相关论文列表</a>
No 46. <a href="https://weibo.com/1402400261/O67DvhBsP">[LG]《Towards Principled Representation Learning from Videos for Reinforcement Learning》</a>
No 47. <a href="https://weibo.com/1402400261/O67F1hNXq">通过prompt-in-decoder(PID)方法，实现输入编码复用和并行解码，显著提高了Transformer的编解码器模型在多输出任务上的训练和推理效率。//[CL]《Encode Once and Decode in Parallel: Efficient Transformer Decoding》</a>
No 48. <a href="https://weibo.com/1402400261/O5Guy1JXq">专家混合(MoE)模型详解</a>
No 49. <a href="https://weibo.com/1402400261/O5qlKiXsS">AutoDev：自动代码生成器，支持多语言的 AI 代码生成，提供自动代码生成、调试和文档功能，帮程序员提高效率</a>
No 50. <a href="https://weibo.com/1402400261/O67Cn1q64">[CL]《Reverse Training to Nurse the Reversal Curse》</a>
